File: loianeTraining/javaBasico/Exer Aula 16_17/Ex41.java
import java.util.Scanner;

public class Ex41 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.println("Digite o valor da divida");
        var debt = scan.nextInt();

        int[] installments = { 1, 3, 6, 9, 12 };
        double[] fees = { 1, 1.1, 1.15, 1.2, 1.25 };
        var totalDebt = 0.0;
        System.out.println("Valor da Divida \t| Valor dos Juros \t| Quantidade de Parcelas \t| Valor da Parcela");
        for (int i = 0; i < 5; i++) {
            totalDebt = debt * fees[i];
            System.out.print("R$ " + (int) totalDebt + "\t\t\t");
            System.out.print((totalDebt - debt) + "\t\t\t");
            System.out.print(installments[i] + "\t\t\t\t");
            System.out.println("R$ " + totalDebt / installments[i] + "\n\n");
        }
        scan.close();
    }
}

File: loiane-training-csharp/procedural-poo/d/Ex41.cs
using System;

class Ex41
{
    static void Main(string[] args)
    {
        Console.Write("Digite o valor de uma divida: ");
        double valorDivida = double.Parse(Console.ReadLine());

        double parcelas = 1;
        Console.WriteLine("Valor da divida \t Valor dos juros \t Quantidade de parcelas \t Valor da parcela");
        for (int i = 0, juros = 50; i < 5; i++, juros += 50)
        {
            if (i != 0)
            {
                parcelas += i == 1 ? 2 : 3;
            }
            double valorParcela = (valorDivida + juros) / parcelas;

            double totalAPagar = valorDivida + juros;
            Console.WriteLine("R$ {0} \t\t {1} \t\t\t {2} \t\t\t\t R$ {3}", totalAPagar, juros, parcelas, valorParcela);
        }
    }
}
